mods/ra/rules/defaults.yaml

```diff
@@ -479,7 +491,13 @@
 	Selectable:
 		Bounds: 24,24
 	Targetable:
-		TargetTypes: Ground, Water, Ship, Repair
+		TargetTypes: Ground, Water, Ship
+	Targetable@REPAIR:
+		TargetTypes: Repair
+		RequiresCondition: !undamaged
+	GrantConditionOnDamageState@UNDAMAGED:
+		Condition: undamaged
```

Comment:
IMO it would be clearer if we could drop the double negatives by defining these as `damaged` conditions, with `ValidDamageStates: Light, Medium, Heavy, Critical`